### ISSUE TITLE
Add end-to-end encryption support to File Provider virtual files implementation

### ISSUE THREAD

**Issue by claucambra** [nextcloud/desktop#6194]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you want to have the same feature implemented.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Feature description
Currently, the File Provider VFS implementation on macOS just ignores end-to-end encrypted files. We should support these fully in the future

**Comment by Tobix99** [comment]
Hi, sorry to ask, but isn't this the same as this Issue?

https://github.com/nextcloud/desktop/issues/5696

**Comment by claucambra** [comment]
> Hi, sorry to ask, but isn't this the same as this Issue?
> 
> #5696

You're right, should've checked more closely before opening this one! closing
